front/informacoes_turma.html

```diff
@@ -134,7 +134,7 @@
                 const mediaAluno = document.createElement("div");
                 mediaAluno.className = "media";
                 mediaAluno.type = "number";
-                mediaAluno.innerHTML = "Media"
+                mediaAluno.innerHTML = await obter_fee_turma_aluno(obter_id(),alunoId)
```

Comment:
Não é melhor jogar isso para logo após o for, atribuindo em um variável chamada FEEAlunoNaTurma, depois vc realizar um innerHtml inserindo essa variável? Pq ta confuso vc ter uma chamada de função async dentro de um innerHTML